lib/player/fake_player.js

```diff
@@ -6,17 +6,11 @@ module.exports = class FakePlayer {
   }
 
   async play(url) {
-    return new Promise(resolve => {
-      this._currentStationUrl = url
-      resolve()
-    })
+    this._currentStationUrl = url
```

Comment:
The reason I added these promises is so that the fake player behaves the same way as the real one - i.e. you need to use `await` when you call the method or it won't work. In other words to make the fake more realistic. Do you think that's unneccesary?

Reply:
When you declare a function as `async` the return value is always a promise. There is no need to return an explicit one.